### PR TITLE
Update CSS for Drodown and add test cases

### LINKED ISSUE #924: [BUG] Background color of a disabled select field to be #F0F0F0
**Describe the bug**
The background color of a disabled select field to be #F0F0F0 instead of #F9F9F9

**To Reproduce**
1. Go to 'https://tangoe.design/ui-kit/forms/select'
2. Scroll down to 'Disable Component'
4. See the background color css

**Expected behavior**
The background color CSS should be **#F0F0F0**

**Screenshots**
image

image

### PR DESCRIPTION
**Pull request by amit-patra**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] The commit message follows our [guidelines](https://github.com/mobi/goponents/blob/main/CONTRIBUTING.md)
- [ ] Tests for the changes have been added
- [ ] Docs have been added or updated

## PR Type
What kind of change does this PR introduce?

- [x] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [x] Style Update (CSS)
- [ ] Other... Please describe:

## What is the current behavior?
The disabled select has a background color of #F9F9F9

Resolves #924 

## What is the new behavior?
The disabled select background color has been changed to #F0F0F0

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: projects/go-lib/src/lib/components/go-select/go-select.component.spec.ts
Comment (sudheepdivakargithub) [inline projects/go-lib/src/lib/components/go-select/go-select.component.spec.ts:1]: I dont see any relevant test case added. 
In this case its advisable to assert that the correct background color is applied while the control is enabled and disabled.

Comment (amit-patra) [reply projects/go-lib/src/lib/components/go-select/go-select.component.spec.ts:1]: I have added a few CSS including disabled test cases as well.

File: projects/go-lib/src/lib/components/go-select/go-select.component.spec.ts [hunk 1]
```diff
@@ -206,4 +226,32 @@ describe("GoSelectComponent", () => {
       fixture.debugElement.queryAll(By.css("go-form-errors"))?.length
     ).toBe(1);
   });
+
```
Comment (sudheepdivakargithub) [inline projects/go-lib/src/lib/components/go-select/go-select.component.spec.ts:229]: Are these tests required? I dont think they are anyway related to the change being made.

Comment (sudheepdivakargithub) [reply projects/go-lib/src/lib/components/go-select/go-select.component.spec.ts]: Code removed. Hence resolving.

**Review by sudheepdivakargithub — APPROVED** [review]

**Pull request merged**